tests/test_flow/test_persistence_aip.py

```diff
@@ -103,7 +103,6 @@ def x_plus_one(x):
 
     log_checker.expect_regex(
         r"Staging AI Platform task .* at gs://.*bionic_x_plus_one.*",
-        r"Submitting AI Platform task on .*x_plus_one.*x=1.*",
```

Comment:
Was this regex incorrect?

Reply:
Nope, it was correct, just mostly redundant. It was targeting the same message as the one on line 108, and the new implementation of `expect_regex` doesn't allow two regexes to match the same messag.

Reply:
Oh, good catch. It may have been a duplicate.